JS/localStorage.js

```diff
@@ -0,0 +1,172 @@
+  </div>
+  <button data-modal-target="#modal" class="btn btn-style btn-style-btn " onclick="showModal('${id}')" id="" href="${linkVersion}" target="_blank">${linkVersion}</button>
+</div>
+
+  `;
+
+  return container;
+};
+
+const renderPopupps = () => {
+  const container = document.querySelector('.container-block');
+
+  const popuppsElement = document.createElement('div');
+  const containerFull = document.createElement('div');
+  containerFull.classList.add('container-full');
+  popuppsElement.appendChild(containerFull);
+  popuppsElement.classList.add('container-form');
+
+  popupps.forEach((popupp) => {
+    containerFull.appendChild(renderPopupp(popupp));
+    container.appendChild(popuppsElement);
+  });
+};
+renderPopupps();
+
+const showModal = (id, isMultipost) => {
+  const modal = document.getElementById('myModal');
+  modal.style.display = 'block';
+  if (isMultipost) {
+    document.getElementById('model-dynamic-content').innerHTML = `
+    <div class='pupopp-item'>
+    <img src="./imgs/Img_Placeholder.svg" alt="here's placeholder image where the alternatives text goes">
+  <div class="section-art sm-screen">
+      <h2>Multi-Post Stories</h2>
+      <p>
+      A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
+      <div class="luang">
+          <ul>
+          <li>CSS</li>
+          <li>html</li>
+          <li>bootstrap</li>
+          <li>Ruby</li>
+          </ul>
+      </div>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " href="" target="_blank">See Live</a>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " href="" target="_blank">See Source</a>
+  </div>
+  </div>
+    `;
+  } else {
+    const item = popupps.find((p) => p.id.toString() === id);
+    const {
+      name, description, featuredImage, technologies, linkVersion,
+    } = item;
+
+    document.getElementById('model-dynamic-content').innerHTML = `
+  <div class='pupopp-item'>
+  <div data-id=${id} class='pupopp lg-screen pupopp-form${id}' style="background:url(${featuredImage}); width: 97vh;"></div>
+  <div class="section-art sm-screen">
+      <h2>${name}</h2>
+      <p>
+      ${description}
+      </p>
+      <div class="luang">
+          <ul>
+          ${technologies.map((technology) => `<li>${technology}</li>`).join('')}
+          </ul>
+      </div>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " onclick="showModal('${id}')" id="" href="${linkVersion}" target="_blank">${linkVersion}</a>
+      <a data-modal-target="#modal" class="btn btn-style btn-style-btn " onclick="showModal('${id}')" id="" href="${linkVersion}" target="_blank">${linkVersion}</a>
+  </div>
+  </div>
+  `;
+  }
+};
+
+const closeModal = () => {
+  const modal = document.getElementById('myModal');
+  modal.style.display = 'none';
+};
```

Comment:
- [ ] On the desktop version we click or tap the **close (X) button,** and the popup disappear. Good job so far :clap: But it is not working for the mobile version please fix it to give your user a great user experience :smile: